packages/db-migrations/databases/fxa/patches/patch-138-139.sql

```diff
@@ -0,0 +1,30 @@
+-- Create carts table to store checkout Cart state when creating a subscription.
+-- An FxA user can have more than one Cart record e.g. if they have more than
+-- one subscription where each subscription was created in a different session.
+-- A Cart record may not be associated with an FxA user e.g. if they started
+-- the new user checkout flow but did not get to the point where we create an
+-- account.
+
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('138');
+
+CREATE TABLE IF NOT EXISTS carts (
+    id BINARY(16) NOT NULL PRIMARY KEY,
+    uid BINARY(16),
+    state ENUM('start', 'processing', 'success', 'fail') NOT NULL,
+    errorReasonId VARCHAR(255),
+    offeringConfigId VARCHAR(255) NOT NULL,
+    `interval` VARCHAR(255) NOT NULL,
+    experiment VARCHAR(255),
+    taxAddress JSON,
+    createdAt BIGINT UNSIGNED NOT NULL,
+    updatedAt BIGINT UNSIGNED NOT NULL,
+    couponCode VARCHAR(255),
+    stripeCustomerId VARCHAR(32),
+    email VARCHAR(255),
+    amount INT UNSIGNED NOT NULL,
```

Comment:
Added these 3 per our meeting this afternoon -- see [notes](https://docs.google.com/document/d/1hUx-lbymI8It_Cn5AeUkVmwdc2_pqCJDqs8G9X6ia8c/edit#bookmark=id.xt28hdabrmhj). The referenced doc will be inlined into the [SubPlat 3.0 Technical Specification](https://docs.google.com/document/d/1nvBlliXSufdMZnSZMfSu7TUrOslQLBXlcHfWoYvrAAU/edit?usp=sharing) for the Cart shortly by @StaberindeZA (thank you!).